src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1531,15 +1531,28 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 					VariableTypeLoader.BuiltInTypes.NUMBER.getName() :
 					VariableTypeLoader.BuiltInTypes.LOGIC.getName());
 			return gamerule;
+		} else if (ModElementType.VILLAGERPROFESSION.equals(modElement.getType())) {
+			VillagerProfession profession = new VillagerProfession(modElement);
+			profession.displayName = modElement.getName();
+			MItemBlock poi = new MItemBlock(modElement.getWorkspace(),
+					getRandomMCItem(random, blocks).getName());
+			profession.pointOfInterest = poi;

```

Comment:
You could inline the `poi` variable here.

Reply:
Done